tests/components/sql/test_sensor.py

```diff
@@ -214,7 +204,6 @@ async def test_invalid_url_on_update(
         )
         await hass.async_block_till_done()
 
-    assert "sqlite://homeassistant:hunter2@homeassistant.local" not in caplog.text
```

Comment:
Has been removed due to the verbose logging from `recorder_mock` so it can not assert